### ISSUE TITLE
Save settings in Chrome Storage

### ISSUE THREAD

**Issue by arkon** [richgilbank/Scratch-JS#69]
It seems like the code is saved in synced Chrome storage, yet the settings are saved in localStorage. Seems like an odd choice.

**Comment by richgilbank** [comment]
Yeah, there were some fairly serious bugs with Chrome storage at the time settings were added. Could definitely use some unification there.

**Comment by abhisekp** [comment]
@richgilbank A save button in the UX and `CTRL+S` keybinding could go along with this.
